### PR TITLE
Cor 93 add user notifications

### PR DESCRIPTION
**Pull request by nilueps**
Notifications are working now for most successful actions.
For some reason, notifs don't show up on Attribute updating, only creating, if one of you can take a look, maybe a fresh pair of eyes will spot the culprit.

### CODE REVIEW

**Comment by keth** [comment]
> Notifications are working now for most successful actions.
> For some reason, notifs don't show up on Attribute updating, only creating, if one of you can take a look, maybe a fresh pair of eyes will spot the culprit.

Is sounds like this is something we could catch with Unit-Tests

**Review by keth — APPROVED** [review]

File: api/pkg/apps/webapp/attributes.go [hunk 1]
```diff
@@ -67,18 +68,18 @@ func (h *Server) Attribute(w http.ResponseWriter, req *http.Request) {
...
-	a, err := iamClient.Attributes().Get(ctx, id)
+	attribute, err := iamClient.Attributes().Get(ctx, id)
 	if err != nil {
 		http.Error(w, err.Error(), http.StatusInternalServerError)
 		return
 	}
 
 	if req.Method == "POST" {
-		h.PostAttribute(ctx, a, w, req)
+		h.PostAttribute(ctx, attribute, w, req)
```
Comment (ludydoo) [inline api/pkg/apps/webapp/attributes.go:78]: return here ?

File: api/pkg/apps/webapp/templates/casetype.gohtml [hunk 2]
```diff
@@ -57,6 +57,10 @@
                         </select>
                         <label for="team">Select Team</label>
                     </div>
+                    {{if .CaseType}}
```
Comment (ludydoo) [inline api/pkg/apps/webapp/templates/casetype.gohtml:60]: whats that for?

File: api/pkg/apps/webapp/attributes.go (hunk 1 above)
Comment (nilueps) [reply api/pkg/apps/webapp/attributes.go:78]: good catch ;)

File: api/pkg/apps/webapp/templates/casetype.gohtml (hunk 2 above)
Comment (nilueps) [reply api/pkg/apps/webapp/templates/casetype.gohtml:60]: values from the disabled selects above we're not getting through

**Comment by ludydoo** [comment]
That doesn’t seem right

Le mer. 30 juin 2021 à 14:05, Nicolas Epstein ***@***.***> a
écrit :

File: api/pkg/apps/webapp/templates/casetype.gohtml (hunk 2 above)
Comment (nilueps) [reply api/pkg/apps/webapp/templates/casetype.gohtml:60]: maybe it was something to do with the values not being passed along properly, I didn't take the time to investigate to be honest.

looking into it now

**Comment by nilueps** [comment]
Ok it turns out disabled input elements are not submitted with forms as I suspected
https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/disabled

**Comment by ludydoo** [comment]
Then read only attribute ?

Le mer. 30 juin 2021 à 14:16, Nicolas Epstein ***@***.***> a
écrit :

**Comment by nilueps** [comment]
yeah

**Comment by nilueps** [comment]
rememeber, we're using the same template for new casetypes as for viewing created ones. so if `.CaseType` if present, we chose to disabled those inputs

**Pull request merged**